web/src/pages/welcome.tsx

```diff
@@ -0,0 +1,12 @@
+import Home from "@/pages/index";
+import { ReactElement } from "react";
+
+const WelcomePage = (): ReactElement => {
+  return (
+    <>
+      <Home isWelcomePage={true} />
+    </>
+  );
+};
```

Comment:
This is a nit-picking point, but you don't have to wrap a stand-alone element in fragment tags. In fact, `<Home>` could just be on the same line as the return, no parentheses.

Reply:
Yes, I had to add them when I added the Welcome string to test functionality and forgot to remove them. Thanks!